examples/worlds/advanced_lift_drag_system.sdf

```diff
@@ -0,0 +1,681 @@
+                    </mesh>
+                    </geometry>
+                    <material>
+                        <ambient>.175 .175 .175 1.0</ambient>
+                        <diffuse>.175 .175 .175 1.0</diffuse>
+                    </material>
+                </visual>
+                <gravity>1</gravity>
+                <velocity_decay/>
+                <self_collide>0</self_collide>
+            </link>
+
+            <joint name='rotor_puller_joint' type='revolute'>
+                <child>rotor_puller</child>
+                <parent>base_link</parent>
+                <axis>
+                    <xyz>1 0 0</xyz>
+                    <limit>
+                        <lower>-1e+16</lower>
+                        <upper>1e+16</upper>
+                    </limit>
+                    <dynamics>
+                        <spring_reference>0</spring_reference>
+                        <spring_stiffness>0</spring_stiffness>
+                    </dynamics>
+                    <use_parent_model_frame>1</use_parent_model_frame>
+                </axis>
+            </joint>
+
+            <link name="left_elevon">
+                <inertial>
+                    <mass>0.00000001</mass>
+                    <inertia>
+                    <ixx>0.000001</ixx>
+                    <ixy>0.0</ixy>
+                    <iyy>0.000001</iyy>
+                    <ixz>0.0</ixz>
+                    <iyz>0.0</iyz>
+                    <izz>0.000001</izz>
+                    </inertia>
+                    <pose>0 0.3 0 0.00 0 0.0</pose>
+                </inertial>
+                <visual name='left_elevon_visual'>
+                    <pose>0.07 0.0 -0.08 0.00 0 0.0</pose>
+                    <geometry>
+                    <mesh>
+                        <scale>0.1 0.1 0.1</scale>
+                        <uri>model://rc_cessna/meshes/left_aileron.dae</uri>
+                    </mesh>
+                    </geometry>
+                    <material>
+                        <ambient>1 0 0 1.0</ambient>
+                        <diffuse>1 0 0 1.0</diffuse>
+                    </material>
+                </visual>
+            </link>
+
+            <link name="right_elevon">
+                <inertial>
+                    <mass>0.00000001</mass>
+                    <inertia>
+                    <ixx>0.000001</ixx>
+                    <ixy>0.0</ixy>
+                    <iyy>0.000001</iyy>
+                    <ixz>0.0</ixz>
+                    <iyz>0.0</iyz>
+                    <izz>0.000001</izz>
+                    </inertia>
+                    <pose>0 -0.3 0 0.00 0 0.0</pose>
+                </inertial>
+                <visual name='right_elevon_visual'>
+                    <pose>0.07 0.0 -0.08 0.00 0 0.0</pose>
+                    <geometry>
+                        <mesh>
+                            <scale>0.1 0.1 0.1</scale>
+                            <uri>model://rc_cessna/meshes/right_aileron.dae</uri>
```

Comment:
We don't have this mesh as part of the default installation path. This means I get the error:
```
[GUI] [Err] [SystemPaths.cc:425] Unable to find file with URI [model://rc_cessna/meshes/body.dae]
[GUI] [Err] [SystemPaths.cc:525] Could not resolve file [model://rc_cessna/meshes/body.dae]
```
while loading this example.

One option is to upload the plane to [fuel](https://app.gazebosim.org/dashboard). The other option is to use simple primitive box shapes in the example. To see an example take a look at the [following lrauv](https://github.com/gazebosim/gz-sim/blob/5d1c5058fba0dc6b19eaf29d2782b49abb36e630/examples/worlds/multi_lrauv_race.sdf#L158) based example: